packages/angular_devkit/schematics/src/tree/host-tree.ts

```diff
@@ -98,6 +96,7 @@ export class HostTree implements Tree {
   private _id = _uniqueId++;
```

Comment:
I don’t see this part being fixed from my previous PR; IDs need to be negative to avoid ID clash with VirtualTree.

Reply:
Done.